pygyro/advection/test_advection.py

```diff
@@ -253,15 +277,82 @@ def test_poloidalAdvection(dt, v, xc, yc):
     finalPts[1][:] = np.sqrt(x * x + y * y)
     final_f_vals[:, :] = initConds(finalPts[1], finalPts[0])
 
-    l2 = np.sqrt(trapz(trapz((f_vals-final_f_vals)**2,
-                 eta_grids[1], axis=0)*eta_grids[0], eta_grids[0]))
+    l2 = np.sqrt(trapz(trapz((f_vals - final_f_vals)**2,
+                 eta_grids[1], axis=0) * eta_grids[0], eta_grids[0]))
+    assert(l2 < 0.2)
+
+
+@pytest.mark.serial
+@pytest.mark.parametrize("dt,v,xc,yc", [(1.0, -5.0, 0, 0), (0.1, 5.0, 1, 0), (0.1, 2.0, 2, 1)])
+def test_poloidalAdvectionArakawa(dt, v, xc, yc):
+    """
+    TODO
+    """
+
+    npts = [64, 64]
+    eta_vals = [np.linspace(0, 20, npts[1], endpoint=False), np.linspace(0, 2*np.pi, npts[0], endpoint=False),
+                np.linspace(0, 1, 4), np.linspace(0, 1, 4)]
+
+    N = int(1 / dt)
+
+    f_vals = np.ndarray([npts[1], npts[0]])
+    final_f_vals = np.ndarray([npts[1], npts[0]])
+
+    deg = 3
+    omega = 1
+
+    domain = [[1, 14.5], [0, 2*np.pi]]
+    periodic = [False, True]
+    nkts = [n + 1 + deg * (int(p) - 1) for (n, p) in zip(npts, periodic)]
```

Comment:
we don't need the spline functionality in the following

Reply:
yes, in the Arakawa tests of `test_advection` we can take out the spline functions

Reply:
We can also do this in the future :+1: